asistente_ladm_col/gui/reports/reports.py

```diff
@@ -340,13 +315,17 @@ def generate_report(self, db, report_type):
             self.logger.warning_msg(__name__, msg)
 
     def download_java_complete(self):
-        self.logger.info_msg(__name__, QCoreApplication.translate("ReportGenerator",
-                                                                  "Java was successfully configured!"), 5)
+        if self.java_dependency.check_if_dependency_is_valid():
+            self.logger.info_msg(__name__, QCoreApplication.translate("ReportGenerator",
+                                                                      "Java was successfully configured!"), 5)
+        else:
+            self.logger.info_msg(__name__, QCoreApplication.translate("ReportGenerator",
+                                                                      "Java dependency not was download!"), 5)
 
     def download_report_complete(self):
-        self.logger.info_msg(__name__, QCoreApplication.translate("ReportGenerator",
-                                                                  "Report dependency was successfully configured!"), 5)
-
-    def download_report_dependency(self):
-        if not self.report_dependency.check_if_dependency_is_valid():
-            self.report_dependency.download_dependency(URL_REPORTS_LIBRARIES)
+        if self.report_dependency.check_if_dependency_is_valid():
+            self.logger.info_msg(__name__, QCoreApplication.translate("ReportGenerator",
+                                                                      "Report dependency was successfully configured!"), 5)
+        else:
+            self.logger.info_msg(__name__, QCoreApplication.translate("ReportGenerator",
+                                                                      "Report dependency not was download!"), 5)
```

Comment:
was not...

Reply:
Thanks, when user cancels the download the appropriate message is showed now. c0ceb5d